src/main/java/org/example/ReadingFileAllNumberOfCharacters.java

```diff
@@ -0,0 +1,24 @@
+package org.example;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static java.lang.Character.isDigit;
+import static java.lang.Character.isJavaLetter;
+
+public class ReadingFileAllNumberOfCharacters {
+    public static void printAllNumberOfCharacters(File file) throws IOException {
+        String pathToFile = file.getPath();
```

Comment:
skoro wyciągasz z tego file path to nie lepiej ten path obrać jako parametr wejściowy zamiast file?